octoprint_smsnotifier/__init__.py

```diff
@@ -101,37 +101,34 @@ def _send_txt(self, payload, snapshot=False):
 
         for number in self._settings.get(['recipient_number']).split(','):
             tonumber = phonenumbers.format_number(phonenumbers.parse(number, 'US'), phonenumbers.PhoneNumberFormat.E164)
-        tags = {
-            'filename': filename,
-            'elapsed_time': elapsed_time,
-            'printer_name': self._settings.get(["printer_name"])
-        }
-        message = self._settings.get(["message_format", "body"]).format(**tags)
-
-        client = TwilioRestClient(self._settings.get(['account_sid']), self._settings.get(['auth_token']))
-        if snapshot:
-            try:
-                client.messages.create(to=tonumber, from_=fromnumber, body=message, media_url=snapshot)
-            except Exception as e:
-                # report problem sending sms
-                self._logger.exception("SMS notification error: %s" % (str(e)))
-                return False
+            tags = {
+                'filename': filename,
+                'elapsed_time': elapsed_time,
+                'printer_name': self._settings.get(["printer_name"])
+            }
+            message = self._settings.get(["message_format", "body"]).format(**tags)
```

Comment:
actually, `tags` and `message` can be done outside the loop too.

Reply:
Agreed -- would be more efficient than doing it every loop iteration.